packages/ui-components/src/lib/storesGeneric/cachedWritableStore.ts

```
@@ -0,0 +1,71 @@
+	cachedWritableStore<string>(
+		key,
+		defaultValue,
+		(v) => v,
+		(v) => v
+	);
+export const cachedWritableInt = (key: string, defaultValue = 0) =>
+	cachedWritableStore<number>(
+		key,
+		defaultValue,
+		(v) => v.toString(),
+		(v) => parseInt(v)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)